cumulusci/core/config.py

```diff
@@ -946,15 +965,19 @@ def create_org(self):
         if not self.namespaced:
             namespaced = ' -n'
 
+        days = ''
+        if self.days:
+            days = ' --durationdays {}'.format(self.days)
+
         alias = ''
         if self.sfdx_alias:
-            alias = ' -a "{}"'.format(self.sfdx_alias)
+            alias = ' -a {}'.format(self.sfdx_alias)
 
         # This feels a little dirty, but the use cases for extra args would mostly
         # work best with env vars
         extraargs = os.environ.get('SFDX_ORG_CREATE_ARGS', '')
-        command = 'sfdx force:org:create -f {}{}{}{} {}'.format(
-            self.config_file, devhub, namespaced, alias, extraargs)
+        command = 'sfdx force:org:create -f {}{}{}{}{} {}'.format(
```

Comment:
starting to wonder if we should name the fields in the format string...

Reply:
Yep, changed in a new commit